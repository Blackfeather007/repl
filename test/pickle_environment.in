{"cmd": "def f := 42"}

{"env": 0, "pickleTo": "test/a.olean"}

{"unpickleFrom": "test/a.olean"}

{"cmd": "example : f = 42 := by rfl", "env": 1}
