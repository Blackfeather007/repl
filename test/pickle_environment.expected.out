{"sorries": [], "messages": [], "env": 0}
{"sorries": [], "messages": [], "env": 0}
{"sorries": [], "messages": [], "env": 1}
{"sorries": [], "messages": [], "env": 2}
